source/agora/common/Amount.d

```diff
@@ -285,6 +285,22 @@ public struct Amount
         return this.value / other.value;
     }
 
+    /***************************************************************************
+
+        Params:
```

Comment:
Could use some documentation about what the function actually does.

Reply:
Oops